docs/reference/handpose.md

```diff
@@ -0,0 +1,181 @@
+***
+#### .video
+> *Object*. HTMLVideoElement if given in the constructor. Otherwise it is null.
+***
+
+***
+#### .config
+> *Object*. containing all of the configuration options passed into the model.
+***
+
+***
+#### .model
+> *Object*. The Handpose model.
+***
+
+***
+#### .modelReady
+> *Boolean*. Truthy value indicating the model has loaded.
+***
+
+### Methods
+
+***
+#### .predict()
+> A function that returns the results of a single hand detection prediction.
+
+  ```js
+  handpose.predict(inputMedia, callback);
+  ```
+
+📥 **Inputs**
+* **inputMedia**: REQUIRED. An HMTL or p5.js image, video, or canvas element that you'd like to run a single prediction on.
+
+* **callback**: OPTIONAL.  A callback function to handle new hand detection predictions. For example:
+
+  ```js
+  handpose.predict(inputMedia, results => {
+    // do something with the results
+    console.log(results);
+  });
+  ```
+
+📤 **Outputs**
+
+* **Array**: Returns an array of objects describing each detected hand. You can see all of the supported annotation in [the Tensorflow source code](https://github.com/tensorflow/tfjs-models/blob/master/handpose/src/keypoints.ts).
+
+  ```js
+  [
+      {
+        handInViewConfidence: 1, // The probability of a hand being present.
+        boundingBox: { // The bounding box surrounding the hand.
+          topLeft: [162.91, -17.42],
+          bottomRight: [548.56, 368.23],
+        },
+        landmarks: [ // The 3D coordinates of each hand landmark.
+          [472.52, 298.59, 0.00],
+          [412.80, 315.64, -6.18],
+          ...
+        ],
+        annotations: { // Semantic groupings of the `landmarks` coordinates.
+          thumb: [
+            [412.80, 315.64, -6.18]
+            [350.02, 298.38, -7.14],
+            ...
+          ],
+          ...
+        }
+      }
+    ]
+  ```
+
+***
+
+#### .on('predict', ...)
+> An event listener that returns the results when a new hand detection prediction occurs.
+
+  ```js
+  handpose.on('predict', callback);
+  ```
+
+📥 **Inputs**
+
+* **callback**: REQUIRED.  A callback function to handle new hand detection predictions. For example:
+
+  ```js
+  handpose.on('predict', results => {
+    // do something with the results
+    console.log(results);
+  });
+  ```
+
+📤 **Outputs**
+
+* **Array**: Returns an array of objects describing each detected hand as an array of objects exactly like the output of the `.predict()` method described above.
+
+
+## Examples
+
+**p5.js**
+* [Handpose_Image](https://github.com/ml5js/ml5-library/tree/development/examples/p5js/Handpose/Handpose_Image)
+* [Handpose_Webcam](https://github.com/ml5js/ml5-library/tree/development/examples/p5js/Handpose/Handpose_Webcam)
+
+**p5 web editor**
+* [Handpose_Image](https://editor.p5js.org/ml5/sketches/Handpose_Image)
+* [Handpose_Webcam](https://editor.p5js.org/ml5/sketches/Handpose_Webcam)
+
+## Demo
+
+No demos yet - contribute one today!
+
+## Tutorials
+
+No tutorials yet - contribute one today!
+
```

Comment:
We should add in the placeholder for the Data and Model Provenance work that @ellennickles has done for the other features: e.g. https://github.com/ml5js/ml5-library/pull/1047/files#diff-8eef2538dbb4d1c92215a82d44c7fa2cR166

Reply:
Added!